docs: components alignment table (#3430)
lumada-design/hv-uikit-react: docs/overview/status/ProjectStatus.stories.mdx

# Project Status

> Stay up-to-date on the status of project and its development. This can help users of the UI kit understand what features they can currently take advantage of, as well as what they can expect to see in future updates.

Reviewer: ```suggestion
Stay up-to-date on the status of the project and its development. This can help users of the UI kit understand which features they can currently take advantage of, as well as what they can expect to see in future updates.
```

- [Versions](#versions)
- [Supported browsers](#supported-browsers)
- [Packages](#packages)
- [DS alignment](#ds-alignment)

## Versions <a id="versions" />

> Here you can find all supported UI Kit versions, their corresponding Design System and core dependencies versions.  

Reviewer: ```suggestion
Here you can find all the supported UI Kit versions, along with their corresponding Design System and core dependency versions.
```

## Packages <a id="packages" />

At the moment, UI Kit v5.x has not available all v4.x packages. We are actively working toward aligning the missing packages with v5.x and they will be released as soon as they are ready, following our [release cycle](/docs/overview-community-releases--page) rules.

> Bear in mind that v4.x and v5.x packages can't be used simultaneously due to the introduction of the new theming system not compatible with the v4.x components.

Reviewer: ```suggestion
Bear in mind that v4.x and v5.x packages can't be used simultaneously due to the introduction of a new theming system that is not compatible with the v4.x components.
```
